pagopa/pn-frontend
# Feat/pn 2320 privacy tos pa

## Short description
Adds Privacy and ToS page in pn-pa-webapp and fix relative links

## List of changes proposed in this pull request
- Adds Privacy and ToS page in pn-pa-webapp

## How to test
Go to route /privacy-tos and check if privacy policy and terms of service accordions and texts
